django_project/changes/templates/version/detail.html

```diff
@@ -34,7 +33,6 @@ <h1>Entries (all)</h1>
                         <span class="glyphicon glyphicon-download"></span>
                         <span style="font-size: 7pt;">R S T</span>
                     </a>
-                {% endif %}
```

Comment:
Are you sure to remove @sonlinux ?
Anonymous users mustn't be able to see these buttons I think

Reply:
Alright @Gustry 
I was following the permission checks in master branch.
Let me do so